src/oop/object_independence/medium/q2/MainTest.java

```diff
@@ -0,0 +1,217 @@
+package oop.object_independence.medium.q2;
+
+import global.ObjectTest;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MainTest {
+    // Java
+    private final String className = "PhoneWaitList";
+    private final String outputClassName = "TestWaitList";
+    private ObjectTest testClass;
+    private ObjectTest outputClass;
+    private final String firstFieldName = "numAhead";
+    private final String getFirstField = "getNumAhead";
+    private final String setFirstField = "setNumAhead";
+    private final int outputNum1 = 5;
+    private final int outputNum2 = 2;
+
+    @Test
+    @Tag("dependency1")
+    @Tag("dependency2")
+    public void correctTestClass() throws Throwable {
+        Clause[] c = new Clause[]{
+                new StringLiteral("There are " + outputNum1 + " people ahead of you"),
+                new Optional(new StringLiteral(" ")),
+                new NewLine(),
+                new StringLiteral("There are " + outputNum2 + " people ahead of you"),
+                new Optional(new StringLiteral(" "))
+        };
+        Object[][] arguments = {
+                {new String[0], String[].class}
+        };
+        String incorrectMethodOutputMessage = "Your main method in the " + outputClassName + " class does not print the correct output.";
```

Comment:
I think it could be more beneficial to the student if we are more specific to the question. You could say something along the lines of not printing the correctly initialize the objects, modify them, and print the result from their toString method.